web/app/services/algolia.ts

```diff
@@ -287,49 +287,38 @@ export default class AlgoliaService extends Service {
 
   /**
    * Returns FacetRecords for a given index and params.
-   * If the user is the owner, the facets will be filtered by the owner's email.
+   * Sends a non-faceted query to Algolia to get the facets of the entire index.
+   * (We don't yet scope facets to the current facetFilters.)
    */
-  getFacets = task(
-    async (
-      searchIndex: string,
-      params: AlgoliaSearchParams,
-      userIsOwner = false,
-    ): Promise<FacetRecords | undefined> => {
-      let query = params["q"] || "";
-      try {
-        let facetFilters = userIsOwner ? [`owners:${this.userEmail}`] : [];
```

Comment:
This only mattered when there were filters on the /my route. We'll bring this back if that's ever the case.